docs/source/developing/developing.adoc

```diff
@@ -7,6 +7,9 @@
 
 toc::[]
 
+[[developing-overview]]
+== Overview
```

Comment:
LOL, that's one way of fixing it. Just use always 'Overview'. Seems that's the way in the rest of the docs as well, so works for me (and I tried so hard to think about unique headings).